tests/general/KeywordBearTest.py

```diff
@@ -14,3 +22,99 @@
     invalid_files=("test line todo",
                    "test line warNING"),
     settings={"keywords": "todo, warning"})
+
+
+class KeywordBearDiffTest(unittest.TestCase):
+
+    def setUp(self):
+        self.section = Section("")
+        self.section.append(Setting('language', 'python3'))
+        self.section.append(Setting('keywords', '\# TODO, \#todo'))
```

Comment:
if you want to escape `#` so it gets not parsed from `Setting` as a comment, you need to escape it twice, as python already does it once :)
-->
```
'\\# TODO, \\#todo'
```